### ISSUE TITLE
Implement forum WG updates to the website

### ISSUE THREAD

**Issue by chuygarcia92** [CommonsBuild/coordination#838]
## 👇🏽  **What is this issue about? Give as much detail as you can.** 
@kristoferlund had an amazing idea some weeks ago about implementing the Discourse API so that the TEC website can display recent updates by the working groups. The idea would be to start this implementation with the Commons Swarm weekly updates post and start migrating the agendas to the forum.

## ✨  **What does success look like? (i.e., the criteria to meet to call this issue done)**
1. The TEC website is automatically updated with the Commons Swarm updates at first.
2. Stewards start implementing agenda updates in their respective forum threads.
3. The updates on the website become automated and work properly

## 🧍🏻‍♀️🧍🏻‍♂️  **Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)**
Stewards and WG leads.

## 🔗  **Are there links or docs related to this issue?**
forum.tecommons.org/

## 👀  **Who should be in the loop about this issue?**
@Zeptimus
@Tam2140

**Comment by Zeptimus** [comment]
Yeah, this is super cool it will create engagement with the community and more vitality to our website is there anything I could do to make it happen?

**Comment by kristoferlund** [comment]
I'd love to hack away at this! Not sure at the moment how much available time I have but will think that through and get back to you.

**Comment by natenatesoo** [comment]
Any updates?!

**Comment by kristoferlund** [comment]
I unfortunately don't have any spare capacity to develop this in the near term. One requirement for this it to be a good idea is also that there is a steady weekly stream of updates from the working groups on the forum.
